src/SplitPane.js

```diff
@@ -118,6 +120,12 @@ class SplitPane extends React.Component {
           }
           const sizeDelta = isPrimaryFirst ? positionDelta : -positionDelta;
 
+          const pane1Order = parseInt(window.getComputedStyle(node).order);
+          const pane2Order = parseInt(window.getComputedStyle(node2).order);
+          if (pane1Order > pane2Order) {
+              sizeDelta = -sizeDelta;
```

Comment:
Looks like you are trying to mutate a const here...

Reply:
Good catch, sorted.